riocli/deployment/execute.py

```diff
@@ -60,6 +64,10 @@ def execute_command(
     shell is ``/bin/bash``. You can also specify the user using the ``--user``
     option. The default user is ``root``.
 
+    To run the command synchronously,set the --run-async flag to false.
```

Comment:
suggestion: If the default is True, why not call the flag --run-sync and if passed then it would become sync.